brats/data/datasets.py

```diff
@@ -21,14 +22,31 @@ def __init__(self, root: str, transform=None):
         self._files = list(os.scandir(root))
         self._transform = transform
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self._files)
 
-    def __getitem__(self, item):
-        scan = nib.load(self._files[item].path)
+    def __getitem__(self, idx: int) -> typing.Any:
```

Comment:
Generally I see that typing is the one module u import classes from directly. Its good anyway.

Reply:
Can you elaborate? I'm not sure if I understand it correctly 😅